frontend/src/components/task/task-table.js

```diff
@@ -256,8 +256,8 @@ class CustomPaginationActionsTable extends React.Component {
                         </div>
                       </TableCell>
                       <TableCell numeric style={ { padding: 5 } }>
-                        <div style={ { width: 40 } }>
-                          { n.value ? `$ ${n.value}` : '$ 0' }
+                        <div style={ { width: 70, textAlign: 'center' } }>
+                          { n.value ? (n.value === '0' ? 'No bounty added' : `$ ${n.value}`) : 'No bounty added' }
```

Comment:
You need to add the translation strings here. Take a look at our ReactIntl implementation.

Reply:
@alexanmtz I made a new commit.

the screenshot after change:
![image](https://user-images.githubusercontent.com/32790369/77722376-92ac7580-7028-11ea-9b13-75c25a59b6c7.png)